wgsl/index.bs

```diff
@@ -761,6 +761,16 @@ The following table lists the standard set of triggering rules that can be filte
 
     <td>A call to a builtin function computes derivatives, but [=uniformity analysis=] cannot prove that the call occurs in [=uniform control flow=].
 
+    See [[#uniformity]].
+
+  <tr>
+    <td><dfn noexport dfn-for="trigger">subgroup_uniformity</dfn>
+    <td>[=severity/error=]
+    <td>The location of the [=call site=] for any
+        [[#subgroup-builtin-functions|subgroup]] or [[#quad-builtin-functions|quad]] built-in function.
+
+    <td>A call to a subgroup or quad builtin function, but [=uniformity analysis=] cannot prove that the call occurs in [=uniform control flow=].
```

Comment:
Also need to mention the triggering location and description for the `subgroupShuffle*`  requirements on the `delta` values?